finder-app/Makefile

```diff
@@ -0,0 +1,16 @@
+CC := gcc
+objects = writer
+
+ifeq ($(CROSS_COMPILE), aarch64-none-linux-gnu-)
```

Comment:
@krish0706 you can simplify the use of $(CROSS_COMPILE) here, will discuss in class tomorrow.